ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1971,6 +1971,12 @@ router_multicast:
     routing: < true | false >
     multipath: < none | deterministic | "deterministic color" | "deterministic router-id" >
     software_forwarding: < kernel | sfe >
+    rpf:
+      routes:
+        < IPv4 subnet / netmask >:
```

Comment:
Please change the data model to the list of dictionary. 

```yaml
rpf:
  routes:
    - subnet: < ipv4/mask >
       nexthop_ip: < IPv4_network/Mask >
       nexthop_interface: < interface >
       distance: < 1 - 255 >  #optional
```

Reply:
sounds good to me